group_project/assignment_1.py

```diff
@@ -1,5 +1,10 @@
 def first_half(a_string):
-    pass
+    if len(a_string) % 2 == 0:
+       half_str = a_string[:3]
+       return half_str
+    elif len(a_string) % 2 != 0:
+        odd_half_str = a_string[:4]
+        return odd_half_str
 
 
 def test_first_half_with_even_length_string():
```

Comment:
This looks great - My only question how does this work with words of different lengths?